setup.py

```diff
@@ -44,13 +44,13 @@ def run(self):
     name='pylibfdt',
     cmdclass = {'build_py' : build_py},
     use_scm_version=True,
-    setup_requires = ['setuptools_scm'],
+    setup_requires = ['setuptools', 'swig'],
```

Comment:
Do you need the change to setuptools ?

Reply:
AIUI, 'swig' here is just a wrapper package to download and install swig which is not a python package. What happens if the distro version of swig is already installed?

Reply:
I use python 3.12 to verify it on Windows, setuptools is necessary.

Without this PR and don't install swig manually
py -3.12 -m pip install git+https://github.com/devicetree-org/pylibfdt.git@main
![image](https://github.com/user-attachments/assets/ab61c386-5243-4398-8fe1-a334cc1a91a5)


With this PR and don't install swig manually.
- py -3.12 -m pip install git+https://github.com/gguo11837463/pylibfdt.git@gua_pylibfdt_depend_swig
![image](https://github.com/user-attachments/assets/4127f5fd-3c29-409e-9ddb-7190e360f5f3)